specs/interop/token-bridging.md

```diff
@@ -72,26 +101,200 @@ contract OptimismSuperchainERC20 is ERC20, ISuperchainERC20 {
+
+### Cross Chain `transferFrom`
+
+The standard will not include a remotely initialized `transfer` implementation as we think this should be handled directly on the chain where the funds live with the `bridgeERC20()` function. `transferFrom`, on the other hand, might get initialized from a call to a contract that is localized in a different chain from the funds. To be composable, the standard will introduce a `remoteTransferFrom()` function.
+
+Notice that this action is no longer atomic, so the contract will need a special implementation to use it and resume execution once the funds arrive in the chain.
+
+**Approvals**
+
+To enable `remoteTransferFrom()`, we must include a separate `xAllowance` mapping that keeps track of the `chainId`. This allowance should be set locally using a new `xApprove()` function (the naming is open for discussion).
+
+Here is what a possible implementation could look like:
+
+```solidity
+function xApprove(address _spender, uint256 _chainId, uint256 _amount) external returns (bool) {
+  // TODO [research] _chainId wildcard as type(uint256).max (this is very risky)
+  _xAllowances[msg.sender][_spender][_chainId] = _amount;
+  emit XApproval(msg.sender, _spender, _chainId, _amount);
+  return true;
+}
+// this will override the ERC20 view function
+function allowance(address _owner, address _spender) external view returns (uint256) {
+  return allowance(_owner, _spender, block.chainid);
+}
+
+function allowance(address _owner, address _spender, uint256 _chainId) external view returns (uint256) {
+  if (_chainId == block.chainId) return _allowance[_owner][_spender];
+  return _xAllowance[_owner][_spender][_chainId];
+}
+```
+
+It is possible for the standard to also have remote approvals. This feature, however, was discarded as an address might have different owners in different chains (think of smart wallets for instance). Moreover, even if this was not an issue, switching `chainId` is not that relevant UX-wise to include a dedicated function in the token standard.
+
+**`remoteTransferFrom()`**
+
+As already mentioned, the `remoteTransferFrom()` \*\*\*\*will allow using funds on a remote chain on behalf of another party. The function should initiate a call to the `L2ToL2CrossChainMessenger` without burning anything on the origin chain.
+
+To process a `remoteTransferFrom()` on the destination, the standard should include a `relayTransferFrom()` function that checks local allowance and calls `burn()`. Then, it will check if the target lives on the same `chainId` to call `relayAndExecute()` to the corresponding contract.
+
+Finally, the standard should included a `relayAndExecute()` function that might be used to resume execution on the origin chain. This is how the atomicity lost in the `transferFrom()` gets addressed.
+
+A possible implementation for these functionalities would look like this:
+
+```solidity
+function remoteTransferFrom(
```

Comment:
After discussing with @smartcontracts, we think its a good idea to remove the remote transfer from logic from the first iteration of the superchain erc20 standard to simplify things as an MVP. It will reduce the amount of test coverage we need and in the short term a `permit2` like contract could be used to enable cross chain approvals. We can revisit this part of the interface again in the future if people are really asking for it

Reply:
Ok, this makes sense. The tradeoff with permit here is devex. We can upgrade later with more time to test, if we think its worth.

Do you think it makes sense to introduce a predeploy that implements permit2 and/or message relaying for these tokens? This would be a middle ground between having it in the standard and leaving all the work to the interacting contract. 
These predeploys could also be implemented afterwards

Reply:
> Do you think it makes sense to introduce a predeploy that implements permit2 and/or message relaying for these tokens?

We can scope this in future work, for now I'd like to leave this out